draft-ietf-satp-core.md

```diff
@@ -90,9 +90,10 @@ Both the origin and destination networks are assumed to be opaque
 in the sense that the interior constructs of a given network
 is not read/write accessible to unauthorized entities.
 
-The protocol utilizes the asset burn-and-mint paradigm whereby the asset
-to be transferred is permanently disabled or destroyed (burned)
-at the origin network and is re-generated (minted) at the destination network.
+The protocol orchestrates an atomic update of ledgers on two networks for the
+purpose of moving an asset cliam on a ledger on one network to become an 
```

Comment:
The SATP protocol must cover the scenarios where the networks are closed/private.  The signed claims is techno-legal mechanism to ensure that the gateways are performing burning/minting, without any way of verifying if that has been done (because the ledgers are closed/private).

This is akin to a distributed database, where the main coordinator database deletes a record XYZ and then sends a message (claim) to the other N remote databases to delete (create) a record whose recordID is XYZ.

There no way that the coordinator database can see into the other N remote databases (to see if all of them have been honest and have performed the task. Its only later when there is inconsistencies across these databases that the issue will be detected.